Update troubleshooting-theme-uploads (#661)
bigcommerce/dev-docs: docs/stencil-docs/deploying-a-theme/troubleshooting-theme-uploads.md

> * You may upload a maximum of 20 custom themes at a time to the control panel's `Store Design` > `My Themes` section.

Reviewer: ```suggestion
* You may upload a maximum of 20 custom themes at a time to the control panel's `Storefront` > `My Themes` section.
```

Author: good catch

* If you reach this maximum, you can delete custom themes to make room for more uploads.

> * Before uploading, you must package custom themes into a Stencil-specific .zip file format, using Stencil CLI's `stencil bundle` command.

Reviewer: Changing this to a file type name instead of a filename extension.
```suggestion
* Before uploading, you must package custom themes into a Stencil-specific zip file format, using Stencil CLI's `stencil bundle` command.
```

> * A theme's .zip file must be no larger than 50 MB. If your file exceeds that size, please use either a WebDAV or a CDN upload to exclude large static assets.

Reviewer: ```suggestion
* A theme's zip file must be no larger than 50 MB. If your file exceeds that size, please use either a WebDAV or a CDN upload to exclude large static assets.
```

* Generated parsed template files must be less than 1 MB.

## Error codes

> <table>

Reviewer: We should use Markdown for this table instead of HTML.

Author: changed table to markdown format

  </tr>
  <tr>
    <td>TR-500</td>

>     <td>The .zip file is larger than the 50MB limit, or the parsed JSON for templates exceeds the 1MB size limit.</td>

Reviewer: ```suggestion
    <td>The zip file is larger than the 50MB limit, or the parsed JSON for templates exceeds the 1MB size limit.</td>
```

  </tr>
  <tr>
    <td>TR-800</td>

>     <td>There was a problem processing the contained config.json file. <b>Please check the config.json documentation for the required keys and for keys that require values.</b></td>

Reviewer: ```suggestion
    <td>There was a problem processing the contained `config.json` file. <b>Please check the `config.json` documentation for the required keys and for keys that require values.</b></td>
```

  </tr>
  <tr>
    <td>TR-900</td>

>     <td>The contained config.json file is missing the required developer information.</td>

Reviewer: ```suggestion
    <td>The contained `config.json` file is missing the required developer information.</td>
```

> | Issue in processing this theme's thumbnail screenshot (`composed_image`). |

Reviewer: ```suggestion
| Issue in processing this theme's thumbnail screenshot (`composed_image`). |
```

| Issue in processing this theme's full-size screenshot (`desktop_screenshot`). |
| Issue in processing this theme's mobile [screenshot](/stencil-docs/stencil-theme-editor/config-json-metadata) (`mobile_screenshot`). |
| Missing file: This theme does not support the Theme Editor, as it is missing its required `[schema.json]` file. |
| One or more of this theme's screenshots are not image files. |
| Theme is missing a valid thumbnail image (`composed_image`). |
| Theme is missing a valid full-size image (`desktop_screenshot`). |
| Theme is missing a valid mobile image (`mobile_screenshot`). |

> | One or more of this theme's images is not of a supported file type. Valid filetypes are: `.jpg`, `.jpeg`, `.png`, `.gif`. |

Reviewer: ```suggestion
| One or more of this theme's images is not of a supported file type. Valid filetypes are: JPEG, PNG, GIF. |
```

| Thumbnail (`composed_image`) image dimensions are not right. The expected dimensions are 600 x 760 pixels. |
| Full-size (`desktop_screenshot`) image dimensions are not right. The expected dimensions are 2048 x 2600 pixels. |
| Mobile (`mobile_screenshot`) image dimensions are not right. The expected dimensions are 304 x 540 pixels. |
| Too-large image file size for a theme screenshot `composed_image` (thumbnails), `desktop_screenshot`, or `mobile_screenshot`] . The maximum supported size is x, but the file's actual size is y. |

## Workarounds and further info

> If a custom theme does not render properly after you upload and apply it to a storefront, make sure you have created the theme's .zip file using the `stencil bundle` command, on a Mac OS, Linux computer, or virtual machine. Using the `stencil bundle` command will exclude Windows-specific errors that have occurred on some bundles.

Reviewer: ```suggestion
If a custom theme does not render properly after you upload and apply it to a storefront, make sure you have created the theme's zip file using the `stencil bundle` command, on a Mac OS, Linux computer, or virtual machine. Using the `stencil bundle` command will exclude Windows-specific errors that have occurred on some bundles.
```
